client/Components/GameBoard/IdentityCard.jsx

```diff
@@ -159,15 +160,29 @@ class IdentityCard extends React.Component {
             ctx.textAlign = 'center';
             ctx.font = '30px Keyforge';
             this.props.houses.forEach((house, index) => {
-                ctx.fillText(house.toUpperCase(), houseNames[index].x, houseNames[index].y);
-                ctx.strokeText(house.toUpperCase(), houseNames[index].x, houseNames[index].y);
+                ctx.fillText(this.props.t(house).toUpperCase(), houseNames[index].x, houseNames[index].y);
+                ctx.strokeText(this.props.t(house).toUpperCase(), houseNames[index].x, houseNames[index].y);
             });
             this.setState({imageUrl: canvas.toDataURL()});
-
-            this.refs.dummy.parentElement.removeChild(this.refs.dummy);
         });
     }
 
+    findColor(number) {
```

Comment:
...? :D

Reply:
How else will the cards truly discover who they are?